caluma/caluma_analytics/schema.py

```diff
@@ -115,19 +124,41 @@ def resolve_available_fields(obj, info, prefix=None, depth=None, **kwargs):
             key=lambda field: field["id"],
         )
 
+    result_data = graphene.Field(AnalyticsOutput)
+
     @staticmethod
     def resolve_result_data(obj, info):
-        return simple_table.SimpleTable(obj)
+        if obj.is_extraction():
+            return simple_table.SimpleTable(obj)
+        else:
+            return pivot_table.PivotTable(obj)
```

Comment:
```suggestion
        if obj.is_extraction():
            return simple_table.SimpleTable(obj)

        return pivot_table.PivotTable(obj)
```

Not sure if this is a thing in python but in our frontend code we have a rule to avoid else returns